scripts/core/editor3/helpers/tansa.js

```diff
@@ -0,0 +1,201 @@
+import {EditorState, Modifier} from 'draft-js';
+import DiffMatchPatch from 'diff-match-patch';
+
+/**
+ * Create a custom html for Tansa
+ *
+ * @param {EditorState} editorState
+ * @returns {String}
+ */
+export function getTansaHtml(editorState) {
+    const content = editorState.getCurrentContent();
```

Comment:
How can this work if you're calling the function like this inside getHtmlForTansa: `getTansaHtml({ state })`, wouldn't it throw an error about `getCurrentContent` not being defined?

Reply:
On getHtmlForTansa is called getTansaHtml with EditorState as parameter: getTansaHtml(editorState).

Reply:
Yes but you're doing:
```js
const editorState = {state};

getTansaHtml(editorState);
```

 so I guess it's like

```js
getTansaHtml({state})
```

right?

Reply:
Yes, :), the {} were not in the right place